### ISSUE TITLE
ERLANG_NODE not working in latest

### ISSUE THREAD

**Issue by theasp** [rroemhild/docker-ejabberd#132]
I'm not sure if the current tag for latest is intended to be public or not, but it doesn't work with `ERLNAG_NODE`:

```
$ docker run -e "ERLANG_NODE=ejabberd@whatever" rroemhild/ejabberd:latest
...
13:25:59.129 [info] Application lager started on node ejabberd@localhost
13:25:59.143 [info] Application crypto started on node ejabberd@localhost
13:25:59.165 [info] Application sasl started on node ejabberd@localhost
13:25:59.193 [info] Application asn1 started on node ejabberd@localhost
13:25:59.193 [info] Application public_key started on node ejabberd@localhost
```

**Comment by nagua** [comment]
Should be fixed with #131

**Comment by theasp** [comment]
Works for me, thanks!
